app/views/posts/show.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Here is a post for a given id </h1>
\ No newline at end of file
+<div class="post">
+  <div class="post-title">
+    <h2>Post:<%= @post.id %> by <%= @post.author.Name %> </h2>
+    <p>Comments: <%= @post.CommentsCounter %>, Likes: <%= @post.LikesCounter %></p>
+  </div>
+  <h3><%= @post.Title %></h3>
+  <h4><%= @post.Text %></h4>
+  <%@post.five_most_recent_comments.each do |comment|%>
```

Comment:
- [ ] The `five_most_recent_comments` method should not be used here on the show page, here we want to show the post with all the comments, we should use that method on the post index page. Kindly do so